modules/library/coverage/src/test/java/org/geotools/image/ImageWorkerTest.java

```diff
@@ -924,6 +925,7 @@ public void testOpacityAlphaRGBDirect() {
     }
     
     @Test
+    @Ignore ("Due to LibCMS Bug")
```

Comment:
Sorry I thought I backed that out as it should only be on my machine (which is affected)